runtime/compiler/codegen/CodeGenerator.hpp

```diff
@@ -35,6 +35,10 @@ class OMR_EXTENSIBLE CodeGenerator : public J9::CodeGeneratorConnector
 
    CodeGenerator(TR::Compilation *comp) :
       J9::CodeGeneratorConnector(comp) {}
+
+   CodeGenerator() :
+      J9::CodeGeneratorConnector(TR::comp()) {}
+
```

Comment:
Actually I'll delegate to @0xdaryl on this one. I don't see this constructor in the master branch. Not sure what it is for, but `TR::comp()` is rather expensive so I think Daryl will have better gauge of this change.

Reply:
It was introduced in the `snapshot` branch of the openj9-omr repo:
https://github.com/eclipse/openj9-omr/blob/852aba7251a37b785df5282768d2e8907ba4eda7/compiler/codegen/CodeGenerator.hpp#L43-L44

Reply:
Thanks for pointing that. Agreed, change is fine!